src/UrlGenerators/LocalUrlGenerator.php

```diff
@@ -103,4 +103,26 @@ private function isInWebroot()
     {
         return strpos($this->getAbsolutePath(), public_path()) === 0;
     }
+
+    /**
+     * Get the prefix.
+     *
+     * If the prefix and the url are not set, we will assume the prefix
+     * is "storage", in order to point to the default symbolink link.
+     *
+     * Otherwise, we will trust the user has correctly set the prefix and/or the url.
+     *
+     * @return string
+     */
+    private function getPrefix()
+    {
+        $prefix = $this->getDiskConfig('prefix', '');
+        $url = $this->getDiskConfig('url');
+
+        if (! $prefix && ! $url) {
+            return 'storage';
+        }
+
+        return $prefix;
```

Comment:
What if $prefix is not set but $url is? I think you are missing a case here.

Reply:
Since Laravel 5.4 the default configuration doesn't have a `prefix`, but has a `url`. See https://github.com/laravel/laravel/blob/master/config/filesystems.php#L51. This is the happy path actually. 5.3 shipped without the `url`, which required users to manually set that or define a `prefix`. And of course this package supports both options as per the documentation.

Reply:
Ok, sorry, missed that it is being accounted for elsewhere.